expr/seriesaggregators.go

```diff
@@ -215,3 +250,34 @@ func crossSeriesRange(in []models.Series, out *[]schema.Point) {
 		(*out)[i].Val -= mins[i].Val
 	}
 }
+
+func crossSeriesLast(in []models.Series, out *[]schema.Point) {
+	for i := 0; i < len(in[len(in)-1].Datapoints); i++ {
+		dp := in[0].Datapoints[i]
+		for _, s := range in {
```

Comment:
https://github.com/bloomberg/metrictank/blob/29561e8da8fa72e23222ad7513c8a55d654f59f8/expr/seriesaggregators.go#L257-L261

it loops forward, but sets `dp` at each iteration as long as it is a number, so, it would be `last` in the end, but I'll adjust to loop in reverse.